pkgs/top-level/python-packages.nix

```diff
@@ -9928,7 +9928,17 @@ in {
 
   solc-select = callPackage ../development/python-modules/solc-select { };
 
-  solo-python = disabledIf (!pythonAtLeast "3.6") (callPackage ../development/python-modules/solo-python { });
+  solo1-cli = disabledIf (!pythonAtLeast "3.6") (callPackage ../development/python-modules/solo1-cli {
+    # TODO remove this once https://github.com/solokeys/solo1-cli/issues/157 is addressed
+    fido2 = self.fido2.overrideAttrs(x: rec {
+      version = "0.9.3";
+      src = fetchPypi {
+          inherit (x) pname;
+          inherit version;
+          sha256 = "sha256-tF6JphCc/Lfxu1E3dqotZAjpXEgi+DolORi5RAg0Zuw=";
+        };
+      });
+  });
```

Comment:
I'd move this override into the package derivation, otherwise it will probably be ignored when updating the package again.

Reply:
I added this in ff7eefc. Thank you for the suggestion, I fully agree to your assessment!